modules/microshift-oc-by-example-content.adoc

```diff
@@ -3,10 +3,10 @@
 // Uses 'source,bash' for proper syntax highlighting for comments in examples
 
 :_content-type: REFERENCE
-[id="microshift-oc-cli-developer_{context}"]
+[id="openshift-cli-developer_{context}"]
```

Comment:
These IDs need to say microshift or it will break the repo. Our commands are different.